app/templates/components/order-card.hbs

```diff
@@ -9,12 +9,15 @@
   <div class="ui card thirteen wide computer ten wide tablet sixteen wide mobile column">
     <div class="order card main content">
       <div class="right floated author">
+        {{#each-in (group-by 'name' this.allTickets) as |name tickets|}}
```

Comment:
Why? Was `(map-by 'ticket' this.order.attendees)` not working?

Reply:
```

        actual: >
            [object Object]
        stack: >
            Error: Assertion Failed: Cannot call get with 'name' on an undefined object.
                at assert (http://localhost:7357/assets/vendor.js:58582:15)
                at Object._get2 [as get] (http://localhost:7357/assets/vendor.js:37585:72)
                at http://localhost:7357/assets/vendor.js:191800:29
                at Array.forEach (<anonymous>)
                at groupBy (http://localhost:7357/assets/vendor.js:191799:34)
                at http://localhost:7357/assets/vendor.js:28338:29
                at deprecateMutationsInAutotrackingTransaction (http://localhost:7357/assets/vendor.js:77990:9)
                at EmberHelperRootReference.fnWrapper [as fn] (http://localhost:7357/assets/vendor.js:28337:70)
                at http://localhost:7357/assets/vendor.js:69659:38
                at runInAutotrackingTransaction (http://localhost:7357/assets/vendor.js:77962:9)
```

Reply:
That's because tests are not provided with ticket and names. Tests need to be fixed. There is no situation where ticket can be undefined on an attendee in real scenario. Tests have mock objects. Not real ones